vue/pages/apps/index.html

```diff
@@ -142,8 +142,8 @@
                 <div v-for="project in sortApps(projects)" v-if="showProject(project)" class="app panel">
                     <p class="panel-heading has-text-right has-text-weight-bold is-size-4 has-text-grey-darker has-background-white-bis cursor-pointer" v-if="project.is" @click="toggleProject(project)">
                         {{ project.name.toUpperCase() }}
-                        <i v-if="project.collapsed" class="fas fa-angle-down"></i>
-                        <i v-if="!project.collapsed" class="fas fa-angle-left"></i>
+                        <i title="expand" v-if="project.collapsed" class="fas fa-angle-left" style="width: 20px"></i>
```

Comment:
Noticed you're using `width: 20px` inline multiple times. Can this be moved to a CSS file and referred to via a class name?

Reply:
Yes, I could. Typically, I only create a class if the style is something that might be common and reusable. As a developer, I like to see widths and hight in the area I am working instead of a separate style sheet or a css class at the top of the page. Is there a negative size to inline styles other than if abused they can be clutter?

Reply:
A few of my reasons:
- Clutter
- Good design will typically maintain a standard of reusable dimensions, fonts, sizes, colors, etc. 
- Separating page content (HTML) from design (CSS)
- Inline CSS is typically reserved for CSS that is dynamical created or injected by JavaScript

Reply:
- I agree.
- Good point.
- Separating page content (HTML) from design (CSS) is ambiguous at best since some frameworks consider both HTML and CSS design elements and logically speaking I would agree that neither are content.
- I agree that some technologies do work differently, but not sure if it is a standard or documented anywhere. Let me know if I am mistaken.

I will make the change due to the first two arguments. It should bring more consistency, predictability, maintainability to the page.